core/loadpoint.go

```diff
@@ -123,6 +123,12 @@ type LoadPoint struct {
 	chargeTimer api.ChargeTimer
 	chargeRater api.ChargeRater
 
+	// wakeUp
+	activeTimer         *ActiveTimer
```

Comment:
der `activeTimer` bräuchte einen besseren Namen. `chargeStartWaiter` oder sowas?

Reply:
hab ich in wakeUpTimer umbenannt